Item/ItemPressedMash.cs

```diff
@@ -15,6 +16,14 @@ public override string GetHeldItemName(ItemStack itemStack)
             return Lang.GetMatching(Code?.Domain + AssetLocation.LocationSeparator + type + "-" + Code?.Path + "-" + ap);
         }
 
+        public override ItemStack OnTransitionNow(ItemSlot slot, TransitionableProperties props)

```

Comment:
Adding this to the PressedMash item class insures that you’ll always get the appropriate amount of rot out, the same as you would if you had finished pressing it.  Since we are no longer converting the mash before it is removed from the press it helps make sure the right amount ends up in the machine if it rots as well.